src/Form/Model/EnableTwoFactorRequest.php

```diff
@@ -6,32 +6,8 @@
 
 class EnableTwoFactorRequest
 {
-    /**
-     * @var string
-     * @Assert\NotBlank
-     */
-    protected $secret;
-
-    /**
-     * @var ?string
-     * @Assert\NotBlank
-     */
-    protected $code;
-
-    public function __construct(string $secret)
-    {
-        $this->secret = $secret;
-    }
-
-    public function getSecret(): string
-    {
-        return $this->secret;
-    }
-
-    public function setSecret(string $secret): void
-    {
-        $this->secret = $secret;
-    }
+    #[Assert\NotBlank]
+    protected ?string $code = null;
 
     public function getCode(): ?string
```

Comment:
Not sure if we still need it here. We could use a typed public property instead.